src/main/resources/application.properties

```diff
@@ -2,3 +2,10 @@ BASECRM_ACCESS_TOKEN=3cec6c7e60b4030c4dfc08f4007de713adc6417fce0c772522388e9be4e
 AMANDA_ID=920274
 SARA_ID=920276
 BERTA_ID=920275
+BASECRM_DEVICE_UUID=1234567
+spring.jpa.hibernate.ddl-auto: create-drop

```

Comment:
This is ok for early testing but later on every change to database schema should be done by Flyway.